docs/biosample.rst

```diff
@@ -12,31 +12,146 @@ sequencing or a protein fraction from a gradient centrifugation.
 Several instances (e.g. technical replicates) or types of experiments (e.g.
 genomic array as well as RNA-seq experiments) may refer to the same Biosample.
 
+**Data model**
+
+ .. list-table:: Definition of the ``Biosample`` element
+   :widths: 25 25 50 50
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Status
+     - Description
+   * - id
+     - string
+     - required
+     - arbitrary identifier
+   * - dataset_id
+     - string
+     - optional
+     - arbitrary identifier
+   * - individual_id
+     - string
+     - optional
+     - arbitrary identifier
+   * - description
+     - string
+     - optional
+     - arbitrary text
+   * - sampled_tissue
+     - :ref:`rstontologyclass`
+     - required
+     - Tissue from which the sample was taken
+   * - phenotypic_features
+     - :ref:`rstphenotypicfeature`
+     - recommended
+     - List of phenotypic abnormalities of the sample
+   * - taxonomy
+     - :ref:`rstontologyclass`
+     - optional
+     - Species of the sampled individual
+   * - individual_age_at_collection
+     - see text
+     - recommended
+     - Age of the proband at the time the sample was taken
+   * - histological_diagnosis
+     - :ref:`rstontologyclass`
+     - recommended
+     - Disease diagnosis that was inferred from the histological examination
+   * - tumor_progression
+     - :ref:`rstontologyclass`
+     - recommended
+     - Indicates primary, metastatic, recurrent
+   * - tumor_grade
+     - :ref:`rstontologyclass`
+     - recommended
+     - List of terms representing the tumor grade
+   * - tumor_stage
+     - :ref:`rstontologyclass`
+     - recommended
+     - List of terms representing the tumor stage (TNM findings)
+   * - diagnostic_markers
+     - :ref:`rstontologyclass`
+     - recommended
+     - Clinically relevant biomarkers
+   * - procedure
+     - :ref:`rstprocedure`
+     - required
+     - The procedure used to extract the biosample
+   * - hts_files
+     - :ref:`rstfile`
+     - optional
+     - list of high-throughput sequencing files derived from the biosample
+   * - variants
+     - :ref:`rstvariant`
+     - optional
+     - List of variants determined to be present in the biosample
+   * - bool is_control_sample
+     - true or false
```

Comment:
`true or false`  -> `boolean`

Reply:
(blushing) I have changed to boolean :-0

Reply:
I mean, really! (rolls eyes)